held/app/src/main/java/com/held/activity/VerificationActivity.java

```diff
@@ -159,34 +163,39 @@ public void afterTextChanged(Editable editable) {
     }
 
     private void callVerificationApi() {
-        HeldService.getService().verifyUser(mPhoneNo, mPin, new Callback<VerificationResponse>() {
+        HeldService.getService().verifyUser(mAuth,mRegId, mPin,"", new Callback<VerificationResponse>() {
                     @Override
                     public void success(VerificationResponse verificationResponse, Response response) {
-                        if (verificationResponse.isVerified()) {
+                      launchComposeScreen();
+                       /* if (verificationResponse.isVerified()) {
                             PreferenceHelper.getInstance(getApplicationContext()).writePreference(getString(R.string.API_pin), Integer.parseInt(mPin));
+                            Log.i("VerificationActivity", "Responce :" + verificationResponse.toString());
                             DialogUtils.showProgressBar();
-                            callLoginUserApi();
+                            launchComposeScreen();
                         }
-                        DialogUtils.stopProgressDialog();
+                        DialogUtils.stopProgressDialog();*/
                     }
 
                     @Override
                     public void failure(RetrofitError error) {
+
                         DialogUtils.stopProgressDialog();
+                        Log.i("VerifictioActivity","Error"+error.toString());
                         if (error != null && error.getResponse() != null && !TextUtils.isEmpty(error.getResponse().getBody().toString())) {
                             String json = new String(((TypedByteArray) error.getResponse().getBody()).getBytes());
                             UiUtils.showSnackbarToast(findViewById(R.id.root_view), json.substring(json.indexOf(":") + 2, json.length() - 2));
                         } else
                             UiUtils.showSnackbarToast(findViewById(R.id.root_view), "Some Problem Occurred");
-                    }
-                }
 
-        );
+                    }
+                });
     }
 
-    private void callUpdateRegIdApi() {
+    /*private void callUpdateRegIdApi() {
+        launchComposeScreen();
         HeldService.getService().updateRegID(PreferenceHelper.getInstance(this).readPreference(getString(R.string.API_session_token)),
```

Comment:
callUpdateRegIdApi() method just intent to other activity so i did it in callVerificationApi() method by calling launchComposeScreen();